src/graphql/typeDefs/dataTypes/community.graphql

```diff
@@ -0,0 +1,59 @@
+type Community @key(fields: "id") {
+  id: ID!
+
+  "Community's name"
+  name: String!
+
+  "unique community slug."
+  slug: Slug!
+
+  # "The @ name for this community, e.g. @that"
+  # handle: String!
+
+  "Community Short Description"
+  description: String
+
+  "A detailed description of Community"
+  aboutUs: String
+
+  "This communities logo"
+  logo: URL
+
+  "Date and time community created"
+  createdAt: Date
+
+  "Date and time when the community was last updated"
+  lastUpdatedAt: Date
+
+  "Events associated with this community. Default filter, ALL"
+  events(filter: CommunityEventsFilter): [Event]
+
+  "Current status of community, ACTIVE, DRAFT, VERIFY"
+  status: CommunityStatus
+
+  "A verified community, E.g. Microsoft official .NET team"
+  isVerified: Boolean
+
+  "Members following community"
+  followers: [PublicProfile]
+
+  "Community moderators"
+  moderators: [PublicProfile]
```

Comment:
we're gonna have to figure out how to implement this well. because they can really only make changes to their own community.